alfresco-transform-core-aio/alfresco-transform-core-aio-boot/src/main/java/org/alfresco/transformer/AIOController.java

```diff
@@ -26,59 +26,32 @@
-import static org.alfresco.transformer.fs.FileManager.createTargetFileName;
-import static org.alfresco.transformer.util.RequestParamMap.SOURCE_ENCODING;
-import static org.alfresco.transformer.util.RequestParamMap.SOURCE_EXTENSION;
-import static org.alfresco.transformer.util.RequestParamMap.SOURCE_MIMETYPE;
-import static org.alfresco.transformer.util.RequestParamMap.TARGET_EXTENSION;
-import static org.alfresco.transformer.util.RequestParamMap.TARGET_MIMETYPE;
-import static org.alfresco.transformer.util.RequestParamMap.TEST_DELAY;
-import static org.springframework.http.HttpStatus.BAD_REQUEST;
-import static org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR;
-import static org.springframework.http.HttpStatus.OK;
-import static org.springframework.http.MediaType.MULTIPART_FORM_DATA_VALUE;
-
-import java.io.File;
-import java.util.List;
-import java.util.Map;
-import java.util.Arrays;
-import java.util.HashMap;
-
-import javax.servlet.http.HttpServletRequest;
-
 import org.alfresco.transform.client.model.config.TransformConfig;
 import org.alfresco.transform.exceptions.TransformException;
-import org.alfresco.transformer.logging.LogEntry;
+import org.alfresco.transformer.executors.Transformer;
 import org.alfresco.transformer.probes.ProbeTestTransform;
-import org.alfresco.transformer.transformers.Transformer;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.core.io.Resource;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.multipart.MultipartFile;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.alfresco.transform.client.model.Mimetype.MIMETYPE_HTML;
+import static org.alfresco.transform.client.model.Mimetype.MIMETYPE_TEXT_PLAIN;
+import static org.alfresco.transformer.util.RequestParamMap.SOURCE_ENCODING;
+import static org.alfresco.transformer.util.RequestParamMap.TRANSFORM_NAME_PARAMETER;
+import static org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR;
+import static org.springframework.http.HttpStatus.OK;
 
 @Controller
 public class AIOController extends AbstractTransformerController
 {
     private static final Logger logger = LoggerFactory.getLogger(AIOController.class);
 
-    // This property can be sent by acs repository's legacy transformers to force a transform,
-    // instead of letting this T-Engine determine it based on the request parameters.
-    // This allows clients to specify transform names as they appear in the engine config files, for example:
-    // imagemagick, libreoffice, PdfBox, TikaAuto, ....
-    // See ATS-731.
-    @Deprecated
-    private static final String TRANSFORM_NAME_PROPERTY = "transformName";
-

```

Comment:
Has this workaround been moved or removed ... ?

Reply:
Unfortunately Erik's workaround has just been moved rather than removed.

Reply:
Thanks